_data/collection-index.yml

```
@@ -523,3 +523,8 @@
   contact: https://github.com/ElanHasson/devcontainer-features/issues
   repository: https://github.com/ElanHasson/devcontainer-features
   ociReference: ghcr.io/elanhasson/devcontainer-features
+- name: Python+MSSQL+Azure SQL Templates
+  maintainer: Mahdi Setayesh
+  contact: https://github.com/Azure-Samples/python-ms-sql-devcontainer/issues
+  repository: https://github.com/Azure-Samples/python-ms-sql-devcontainer
+  ociReference: ghcr.io/Azure-Samples/python-ms-sql-devcontainer
```

```suggestion
  ociReference: ghcr.io/azure-samples/python-ms-sql-devcontainer
```

The ORAS spec requires the `ociReference` to be in lower case. Can we update here, thanks?
The supporting tools do help convert the case, however, its better to sync as much as possible.